rust/loqui_client/src/connection_handler.rs

```diff
@@ -198,13 +208,13 @@ impl<E: Encoder> ConnectionHandler<E> {
             payload,
         } = response;
         match self.waiters.remove(&sequence_id) {
-            Some(waiter_tx) => {
+            Some(waiter) => {
                 let response = self.config.encoder.decode(
                     transport_options.encoding,
                     is_compressed(flags),
                     payload,
                 );
-                notify_waiter(waiter_tx, response);
+                waiter.notify(response);
             }
             None => {
                 warn!("No waiter for sequence_id. sequence_id={:?}", sequence_id);
```

Comment:
This may happen if the server replies to a request that has exceeded its deadline. Do you think we should warn here or just quietly drop?

Reply:
I suppose I'll change it to debug.